action.yml

```diff
@@ -4,56 +4,41 @@ description: GitHub Action to trigger ci.yml workflow_dispatch event via GitHub
 runs:
   using: composite
   steps:
-
-    - name: Get type
-      id: gettype
+    - name: Derive types
+      id: types
       shell: bash
       env:
         GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         # repository that has a twice-weekly schedule run on consecutive days - use day of year
-        date_part=$(date +%-j)
-        type=""
+        DATE_PART=$(date +%-j)
+        MAJOR_TYPE=""
+        MINOR_TYPE=""
         if [[ $GITHUB_REPOSITORY == "silverstripe/installer" ]] || \
             [[ $GITHUB_REPOSITORY == "silverstripe/recipe-kitchen-sink" ]]
         then
           # respository that has daily schedule - use hour
-          date_part=$(date +%H)
+          DATE_PART=$(date +%H)
         fi
-        if (( $date_part % 2 == 1 )); then
+        if (( $DATE_PART % 2 == 1 )); then
           # current major, next-minor e.g. "5"
-          type=current,next-minor
+          MAJOR_TYPE="current"
+          MINOR_TYPE="next-minor"
         else
           # previous major, next-patch e.g. "4.13"
-          type=previous,next-patch
+          MAJOR_TYPE="previous"
+          MINOR_TYPE="next-patch"
         fi
-        echo "type=$type" >> $GITHUB_OUTPUT
+        echo "MAJOR_TYPE is $MAJOR_TYPE"
+        echo "type=$TYPE" >> $GITHUB_OUTPUT
+        echo "MINOR_TYPE is $MINOR_TYPE"
+        echo "type=$TYPE" >> $GITHUB_OUTPUT
 
-    - name: Check if should dispatch workflow
-      id: checkshoulddispatch
+    - name: Fetch repo branches
       shell: bash
       env:
-        RUN_BRANCHES_ON_PREVIOUS_MAJOR: "false"
```

Comment:
This was the old and crude way to disable support for the previous major. I've updated it to be dynamic based on the existence of a framework y.2.0 tag